### PR TITLE
[ADD] Product Unique Serial setter

### PR DESCRIPTION
**Pull request by r4m-juan**

### CODE REVIEW

File: product_unique_serial_picking/model/stock.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from openerp import _, api, fields, exceptions, models
```
Comment (moylop260) [inline product_unique_serial_picking/model/stock.py]: This file is duplicated with:
https://github.com/Vauxoo/stock-logistics-workflow/blob/8.0/product_unique_serial/model/stock.py

File: product_unique_serial_picking/model/product.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models
+
```
Comment (moylop260) [inline product_unique_serial_picking/model/product.py]: This file is duplicated with:
https://github.com/Vauxoo/stock-logistics-workflow/blob/8.0/product_unique_serial/model/product.py

**Comment by moylop260** [comment]
Could you add depends to main module and create wizard with inherit if is necessary?

**Comment by juanjosepimentel** [comment]
structure changed

**Comment by moylop260** [comment]
Could you fix [lint checks](https://travis-ci.org/Vauxoo/stock-logistics-workflow/jobs/88799719):

``` bash
************* Module product_unique_serial_picking
product_unique_serial_picking/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
************* Module product_unique_serial_picking.__openerp__
product_unique_serial_picking/__openerp__.py:2: [C8101(manifest-required-author), ] Missing author required "Vauxoo" in manifest file
product_unique_serial_picking/__openerp__.py:2: [C8102(manifest-required-key), ] Missing required key "license" in manifest file
pr count_errors 3
```

and odoo unittests [error](https://travis-ci.org/Vauxoo/stock-logistics-workflow/jobs/88799720#L478)

``` bash
ParseError: "External ID not found in the system: product_unique_serial_picking.view_set_product_serial" while parsing /home/travis/build/Vauxoo/stock-logistics-workflow/product_unique_serial_picking/views/stock_view.xml:4, near
```

**Comment by moylop260** [comment]
@juanroute4me 
ping

**Comment by moylop260** [comment]
Worked here https://github.com/Vauxoo/stock-logistics-workflow/pull/11

**Pull request closed without merging**